src/components/common/blocks/gas-price/style.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+import { Card } from '@digix/gov-ui/components/common/common-styles';
+
+export const SliderCard = styled(Card)`
+  height: 10rem;
+  padding: 2.5rem 5rem;
+  background-color: ${props => props.theme.backgroundTertiary.lightest.toString()};
+  box-shadow: none;
+  flex-direction: column;
+  margin-top: 2rem;
+  margin-bottom: 0;
+`;
+
+export const DefaultOption = styled.a`
+  color: #c09f57;
```

Comment:
Let's avoid using hex values. I believe this value is equivalent to `secondaryColor.default`, so the equivalent to this line is:

```js
color: ${props => props.theme.textSecondary.default.toString()};
```

You can view the theme variables in this file: `src/theme/light.js`